example/websocket/client/async-ssl/websocket_client_async_ssl.cpp

```diff
@@ -114,6 +114,16 @@ class session : public std::enable_shared_from_this<session>
         // Set a timeout on the operation
         beast::get_lowest_layer(ws_).expires_after(std::chrono::seconds(30));
 
+        // Set SNI Hostname (many hosts need this to handshake successfully)
+        if(! SSL_set_tlsext_host_name(
+                ws_.next_layer().native_handle(),
+                host_.c_str()))
+        {
+            ec = beast::error_code(static_cast<int>(::ERR_get_error()),
```

Comment:
Do all the examples which call `SSL_set_tlsext_host_name` also call `::ERR_get_error`?

Reply:
Yes.